File: 백준/Bronze/4504. 배수 찾기/배수 찾기.c
#include <stdio.h>

int main(void) { 
   int a, b;
   scanf("%d", &a);
   
   while(1){
      scanf("%d", &b);
      if(b == 0)   break;

      if(b % a != 0)   printf("%d is NOT a multiple of %d.\n", b, a);
      else   printf("%d is a multiple of %d.\n", b, a);
   }
   
   return 0;
}

File: 백준/Bronze/4504. 배수 찾기/배수 찾기.py
a = int(input())

while(True):
  b = int(input())
  if b == 0:
    break
  
  if b % a != 0:
    print(f"{b} is NOT a multiple of {a}.")

  else:
    print(f"{b} is a multiple of {a}.")